src/read.c

```diff
@@ -3390,6 +3394,29 @@ static avifDecoderItem * avifDecoderDataFindItem(avifDecoderData * data, avifBoo
     return NULL;
 }
 
+static avifResult avifDecoderGenerateImageTiles(avifDecoder * decoder, avifTileGroup * tileGroup, avifDecoderItem * item, avifBool alpha)
```

Comment:
The parameter order is consistent with the file, I just noticed there's no this/in/out ordering that's common.